tests/infrastructure/test_azure.py

```
@@ -0,0 +1,1883 @@
+
+        async def mock_ssh_exec(cmd, timeout=300):
+            return 0, "/home/azureuser/.mcpbr_run_12345", ""
+
+        azure_provider._ssh_exec = mock_ssh_exec
+
+        mock_sftp = MagicMock()
+        mock_client.open_sftp.return_value = mock_sftp
+
+        with patch("builtins.open", mock_open(read_data="invalid json")):
+            with patch("pathlib.Path.unlink"):
+                with pytest.raises(Exception):  # JSON decode error
```

Ruff (0.14.14)

[warning] Do not assert blind exception: `Exception`

(B017)